Jenkinsfile.sh

```diff
@@ -17,9 +17,16 @@ export ENV_ARG=$environment
 export RC_DIR_ARG="/opt/rocket-chat"
 export S3_BUCKET_ARG="seekingalpha-rocketchat-builds"
 
+if [[ "$version" == rocket.chat-*.tgz ]]
+then
+  versionized_file="$version"
+else
+  versionized_file="rocket.chat-$version.tgz"
+fi
+
 ## Render Script Templates
 ## Note: $version is a Jenkins job parameter
```

Comment:
Move this comment either to line 20 (where `$version` is used) or to the top of the file (describing its inputs)